lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -136,7 +136,7 @@ def _check_split_test(self, user_tag):
             self.assertIn(visible, content)
 
 
-class TestVertSplitTestVert(SplitTestBase):
+class TestSplitTestVertVert(SplitTestBase):
```

Comment:
That's fair. This tests a unit where a vertical contains a split test, which is used to determine the content of an inner vertical.

The name change wasn't for clarity, the content of the test changed. I think some more informative comments are in order.